loader.go

```diff
@@ -347,11 +349,14 @@ func (l *Loader) loadVPC() (*ec2.Vpc, error) {
 
 // UpsertBucket upserts bucket if it does not exist
 func (l *Loader) UpsertBucket() error {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
looks good, but can you make the timeout a global constant vs setting the value every time?